src/test/java/be/syntra/devshop/DevshopBack/controllers/ProductControllerTest.java

```diff
@@ -188,56 +147,62 @@ void canUpdateProductTest() throws Exception {
                 .andExpect(jsonPath("$.archived").value(productDtoDummy.isArchived()))
                 .andExpect(jsonPath("$.price").value(1.00));
 
-        verify(productService, times(1)).save(productDtoDummy);
+        verify(productService, times(1)).save(any(Product.class));
     }
 
     @Test
     @WithMockUser
-    void retrieveAllProductsFoundBySearchRequestTest() throws Exception {
+    void retrieveAllCategoriesTest() throws Exception {
         // given
-        String searchRequest = "POst";
-        ProductList dummyProductList = productMapperUtility.convertToProductListObject(List.of(createNonArchivedProduct()));
-        when(productService.findAllByNameContainingIgnoreCaseAndArchivedFalse(searchRequest)).thenReturn(dummyProductList);
+        final List<Category> categories = createCategoryList();
+        final CategoryList categoryListDummy = categoryMapperUtility.convertToCategoryList(categories);
+        when(categoryService.findAll()).thenReturn(categoryListDummy);
 
-        // when
+        // then
```

Comment:
yeah, this should still be "when" though 😄

Reply:
fixed